architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,32 +262,58 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
     - created: 2016-01-29T13:40:11Z
       dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
       generation: 1
-      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
+    tag: latest <4>
 ----
+
+<1> The name of the image stream. 
+<2> The image source repository.
+<3> The SHA identifier that this imagestream tag currently references. Resources that reference this imagestream tag use this identifier.
+<4> The image stream tag.
+
 ====
+//Ben: i'm not really sure what purpose status.dockerImageRepository even serves, since each status.tag indicates where the tag comes from. @soltysh ?
```

Comment:
Like I've mentioned elsewhere, we deprecated `.spec.dockerImageRepository`, @bparees did you deprecate the `.status.dockerImageRepository` along with the other? 
Currently it should provide with a pull spec, usually cluster internal, unless there's a public url for the registry configured.

Reply:
no, I didn't deprecate .status.dockerImageRepository but i'm glad you suggested it too because i proposed it to @smarterclayton and he explained to me all the reasons we can never do that (that it presents the target for pushes to the imagestream, ie the internal registry service ip)

Reply:
@bparees @soltysh What does the user set for `.status.dockerImageRepository`?

Reply:
the user doesn't set it, it gets set by the system.  it will point to the internal registry service ip.

Reply:
@bparees hmmm... I'm not sure if that is still true with our current abilities where each tag can point to a totally different backing repo. In which case we still should read that information per tag, imho. It's worth a second thought, I guess.

Reply:
@soltysh yes I raised similar questions to @smarterclayton  but the answer I got is that status.DockerImageRepository is the location where users can push to in the internal registry.  I guess they already know where to push to if they want to update the external image (on dockerhub for example) but if they want to push to the imagestream directly to create new tags, status.DockerImageRepository is the registry host they'd push to to accomplish that.

I'm glad I'm not the only one confused by this, though :)